Components/Decks/DraftDeckEditor.jsx

```diff
@@ -0,0 +1,428 @@
+                                        blankOption={ { label: '- Select -', code: '' } } button={ { text: 'Add', onClick: this.onAddBanner.bind(this) } } />
+                                    <div className='col-sm-9 col-sm-offset-3 banner-list'>
+                                        { banners }
+                                    </div>
+                                </div>
+                            }
+                        </form>
+                    </div>
+                    <div className='col-sm-1' />
+                    <div className='col-sm-6'>
+                        <DeckSummaryHeader
+                            deck={ this.getDeckFromState() }
+                            onCardMouseOut={ () => this.clearHoverCard() }
+                            onCardMouseOver={ card => this.updateHoverCard(card) } />
+                    </div>
+                </div>
+                <div className='row'>
+                    <div className='col-sm-12' style={ { 'text-align': 'center', 'margin-bottom': '1em' } }>
+                        Click cards under the "Remaining Drafted Cards" to move them into your deck and vice-versa.
+                    </div>
+                </div>
+                <div className='row'>
+                    <div className='col-sm-5'>
+                        <div className='form-group'>
+                            <label className='col-sm-12'>Remaining Drafted Cards:</label>
+                        </div>
+                        <div className='col-sm-12'>
+                            <CardTypeGroups
+                                cards={ this.state.remainingCards }
+                                onCardClick={ card => this.handleAddCard(card.code) }
+                                onCardMouseOut={ () => this.clearHoverCard() }
+                                onCardMouseOver={ card => this.updateHoverCard(card) } />
+                        </div>
+                    </div>
+                    <div className='col-sm-1' style={ { 'text-align': 'center' } }>
+                        <span className='glyphicon glyphicon-arrow-left' />
+                        <span className='glyphicon glyphicon-arrow-right' />
+                    </div>
+                    <div className='col-sm-6'>
+                        <div className='form-group'>
+                            <label className='col-sm-12'>Deck:</label>
+                        </div>
+                        <div className='col-sm-12'>
+                            <CardTypeGroups
+                                cards={ this.state.plotCards.concat(this.state.drawCards) }
+                                onCardClick={ card => this.handleRemoveCard(card.code) }
+                                onCardMouseOut={ () => this.clearHoverCard() }
+                                onCardMouseOver={ card => this.updateHoverCard(card) }
+                                useScheme={ this.state.agenda && this.state.agenda.code === '05045' } />
```

Comment:
useScheme is missing an "s" as the prop name in CardTypeGroups is useSchemes

Reply:
Fixed!